packages/happn-db-provider-loki/index.js

```diff
@@ -69,6 +71,7 @@ module.exports = class LokiDataProvider extends commons.BaseDataProvider {
     }
 
     this.settings.snapshotRollOverThreshold = this.settings.snapshotRollOverThreshold || 1e3; // take a snapshot and compact every 1000 records
+    this.settings.fileSizeDifferenceLimit = this.settings.fileSizeDifferenceLimit || 1e7;
```

Comment:
I think we should not do a breaking change here.  But this feature and default to the same operation as before.  i.e. off, and no checking file size on every storePlayback.  Then via configuration we can swich this on.

Reply:
Fixed, default to not checking for this.